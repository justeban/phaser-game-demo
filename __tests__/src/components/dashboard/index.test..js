import React from 'react';
import renderer from 'react-test-renderer';

import Dashboard from '../../../../src/components/dashboard/index.js';

describe('<Dashboad /> (Enzyme Test)', () => {

  it('testing intial state of Dashboard at start', () => {
    let app = shallow(<Dashboard />);
    expect(app.find('h2').exists()).toBeTruthy();
  });

});

describe('<Dashboard /> (Snapshot Test)', () => {

  it('renders cleanly', () => {
    const component = renderer.create(
      <Dashboard />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
