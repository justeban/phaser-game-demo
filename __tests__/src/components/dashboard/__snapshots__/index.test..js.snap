// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dashboard /> (Snapshot Test) renders cleanly 1`] = `
Array [
  <h2>
    Dashboard
  </h2>,
  <form
    onSubmit={[Function]}
  >
    <input
      name="title"
      type="text"
    />
    <input
      name="content"
      type="text"
    />
    <input
      type="submit"
    />
  </form>,
  <ul />,
]
`;
